MLAPI/MonoBehaviours/Core/NetworkedBehaviour.cs

```diff
@@ -921,7 +922,74 @@ private bool SetDirtyness()
             }
             return dirty;
         }
-        
+
+        #endregion
+
+        #region NetworkedVar
+
+        bool networkedVarInit = false;
+        const string networkedVarTypeName = "NetworkedVar`1";
+        List<INetworkedVar> networkedVarFields = new List<INetworkedVar>();
```

Comment:
Use Array

Reply:
I followed the same pattern as SyncVarInit() for this which iterates over all fields and adds ones with SyncedVar attribute to a List<FieldInfo>.

I can reflect through all fields, count the number of NetworkedVars, and then create a fixed size array instead of using List<>. Just curious whether you think this case is special or whether Array should be used in both cases.

Reply:
Array should be used in both cases.